src/views/BillDisplay/BillDisplay.jsx

```diff
@@ -20,7 +20,7 @@ class BillDisplay extends Component {
     const BillsList = (billsVar) => {
       return (
         <div className="bill__container">
-          {billsVar.length > 0 ? (
```

Comment:
why?

this is good

Reply:
its already handled on line 50   bills.length > 0 ? (
            BillsList(bills)
        ) : (
          <p> No Bills Found</p>
        )}